### PR TITLE
Sharing A Project

### PR DESCRIPTION
**Pull request by Cyclostone**
The shared project is a successful project done by Team Venom, And has a lot of potentials to save a lot of lives

### CODE REVIEW

File: snake_bot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Snake Bot
+desc: A Robotic Snake which is used for mainly rescue operations in times of disasters.
+site: No Site yet
```
Comment (ritwik12) [inline snake_bot.yml:3]: @Cyclostone You can give the link to your project repo.

File: snake_bot.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+tags:
+- Snake Robotics
+- Disaster management
+- Snake between crevices
+upforgrabs:
+  name: help needed
+  link: https://github.com/Cyclostone/project/labels/up%20for%20grabs
```
Comment (ritwik12) [inline snake_bot.yml:10]: @Cyclostone This link is invalid

File: snake_bot.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: Snake Bot
+desc: A Robotic Snake which is used for mainly rescue operations in times of disasters.
+site: No Site yet
+tags:
```
Comment (ritwik12) [inline snake_bot.yml:4]: Mention languages used too

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by shiftkey** [comment]
Closing this out due to inactivity.

@Cyclostone once you have a project up and running, feel free to submit a listing again that Up for Grabs can point to.

**Pull request closed without merging**